### ISSUE TITLE
Version number jump?

### ISSUE THREAD

**Issue by levrik** [eamodio/vscode-gitlens#248]
Sorry for this silly question: Why did the version number jump from 7.2.0 directly to 7.5.1?
I don't dare to install the update since it seems a bit fishy to me because there is no explanation somewhere at all.
I already saw that there were multiple 7.3.x betas but where were the 7.4.x and 7.5.0?

**Comment by eamodio** [comment]
Ah, fun with version numbers -- they have so many different meanings and interpretations. I don't really follow any strict version rules (i.e. I largely pick version numbers that "feel" right to me). Although, I usually only bump the patch version (the 3rd number) for patch releases (since bumps in the patch version won't trigger GitLens to show the new version notice). Also for the first 2 numbers, they are definitely more "feel" based. So with regard to 7.5, while I was developing it, it started off as a smaller feature release (so I called it 7.3) and released a few betas. During the development its scope grew so I opted to call it 7.5 (rather than release it as 7.3). And so far 7.5 has had 3 patch releases, so it is now 7.5.3.

Hope that helps.
